src/Http/Http.Extensions/src/HttpRequestJsonExtensions.cs

```diff
@@ -51,36 +44,17 @@ public static class HttpRequestJsonExtensions
     /// <param name="options">The serializer options to use when deserializing the content.</param>
     /// <param name="cancellationToken">A <see cref="CancellationToken"/> used to cancel the operation.</param>
     /// <returns>The task object representing the asynchronous operation.</returns>
-    [RequiresUnreferencedCode(RequiresUnreferencedCodeMessage)]
```

Comment:
I recall @eerhardt mentioning that public ASP.NET APIs should preserve RUC/RDC annotations _even if_ any reflection code is hidden behind a feature flag.